src/components/Template/Template.stories.tsx

```diff
@@ -10,29 +10,31 @@ import Form, { FormField, FormRow } from "../Form/Form";
 import Heading from "../Heading/Heading";
 import Hero from "../Hero/Hero";
 import HorizontalRule from "../HorizontalRule/HorizontalRule";
+import Image from "../Image/Image";
 import Placeholder from "../Placeholder/Placeholder";
 import Table from "../Table/Table";
+import Text from "../Text/Text";
 import TextInput from "../TextInput/TextInput";
 import {
   Template,
   TemplateBreakout,
-  TemplateTop,
+  TemplateFull,
   TemplateMain,
-  TemplateMainNarrow,
   TemplateSidebar,
-  TemplateBottom,
-  sidebarPlacementArray,
+  templateVariantArray,
 } from "./Template";
+import { getPlaceholderImage, sidebarLabel } from "../../utils/utils";
+import useResponsiveSpacing from "../../hooks/useResponsiveSpacing";
 
 const meta: Meta<typeof Template> = {
   title: "Components/Page Layout/Template",
   component: Template,
   argTypes: {
     id: { control: false },
-    sidebar: {
+    variant: {
       control: { type: "radio" },
-      options: sidebarPlacementArray,
-      table: { defaultValue: { summary: "none" } },
+      options: templateVariantArray,
```

Comment:
I'm not sure why, but the `variant` type is showing up as `string` on the Storybook args table.

Reply:
I noticed this a while ago when we first started reworking the `Template` component. The description is missing as well. We talked about it a little bit, but we never got to the bottom of the problem.

Reply:
I don't think so. This was also the case when the prop name was still `sidebar`.